README.md

```diff
@@ -213,12 +213,24 @@ Or to run them manually:
 
 To run benchmarks:
 
-```
+```shell
 make benchmark
 ```
 
 Benchmarks are pushed to the [GitHub pages](https://open-space-collective.github.io/open-space-toolkit-astrodynamics/dev/bench/).
 
+### Validation
+
+OSTk has a cross-validation framework built into it, which allows us to compare its end-to-end propagation accuracy of a full "mission sequence" to other flight qualified tools like GMAT and Orekit.
+
+A standardized input format (a yaml file) to define the particular mission sequence scenario is used, and is tooling agnostic.
+
+The validation framework takes this yaml scenario definition (inside the /test/Validation/data/scenarios folder) and runs the scenario using OSTk. Next, it compares the outputs (things like the position, velocity, mass, acceleration of the spacecraft) at each timestep during the scenario and checks if they are within a certain tolerance of those same quantities generated by GMAT or Orekit in another repo and read in as a .csv file.
+
+Each scenario, the tool it is being compared against, what output quantities are being compared and to what tolerance are defined in the `parameters` variable inside the `Framework.validation.cpp` file, and can be modified or have new scenarios and tolerances appended to them.
```

Comment:
Missing words? 🤔

Reply:
lol yes, wrote it while I was half asleep, can't english